ci/test-conformance-aks.sh

```diff
@@ -206,7 +206,7 @@ function deliver_antrea_to_aks() {
 
     antrea_image="antrea-ubuntu"
     DOCKER_IMG_VERSION=${CLUSTER}
-    DOCKER_IMG_NAME="antrea/antrea-ubuntu"
+    DOCKER_IMG_NAME=""
```

Comment:
this change happened by mistake, because for all the ci related scripts I planned to do that in a separate PR, so now should it be done here @luolanzone